evm-contracts/src/v0.6/dev/ValidatingAggregator.sol

```diff
@@ -0,0 +1,90 @@
+pragma solidity 0.6.6;
+
+
+import './../FluxAggregator.sol';
+import './AnswerValidatorInterface.sol';
+
+
+/**
+ * @title FlaggingAggregator
+ * @notice This contract adds validation capabilities on write to the
+ * FluxAggregator, by allowing a validating contract to be configured.
+ */
+contract ValidatingAggregator is FluxAggregator {
+  AnswerValidatorInterface public answerValidator;
+
+  uint256 private constant VALIDATOR_GAS_LIMIT = 100000;
+
+  event AnswerValidatorUpdated(
+    address indexed previous,
+    address indexed current
+  );
+
+  constructor(
+    address _link,
```

Comment:
Here we again use the `_` prefix convention. I like it for readability, but we should keep consistent with other contracts and decide are we using it or not.